### ISSUE TITLE
Feature Request: New Battery Label - Unknown

### ISSUE THREAD

**Issue by binaryronin** [jaagr/polybar#678]
I have two batteries, and I don't mind having two statuses on my poly bar. The problem is that when one battery is being used, the other is state 'Unknown'. I would like to be able to define this state to be able to show custom animations/output just like the other states (Charging, Discharging, Full). 

Alternatively pull request #227 could also be merged and I could live with a single output that is averaged between the two batteries.

I know those of us with two batteries are few and far between, but right now two battery handling is subpar without using external scripts.

**Comment by patrick96** [comment]
I don't think I follow.
One of your battery modules actually displays `Unknown`?
Can you post the config for that and what does your `/sys/class/power_supply/` look like?

If polybar is not able to get the battery status and is displaying `Unknown`, it is most likely that the issue is with how your batteries report their status. But as far as I can tell, the battery module should never just display `Unkown`, since none of the tokens in the battery module are supposed to display text, but rather numbers, so this is strange behavior altogether.

**Comment by binaryronin** [comment]
I apoligize, that probably wasn't completely clear. It is not Polybar that is showing an unknown state, it is ACPI itself, and I am asking for a way to control for that state within Polybar.

When one battery is in use, it obviously shows either 'Charging' or 'Discharging' states. The second battery however, always shows a state of 'Unknown' since it is literally doing nothing.

```
3643 ◯  acpi
Battery 0: Discharging, 72%, 02:03:45 remaining
Battery 1: Unknown, 5%
```
The states will switch, so when I am using BAT1, BAT0 will show as 'Unknown' state. Simply put, if a battery is not actively in use (either Charging or Discharging) it will show as 'Unknown'.

Does that make sense?

**Comment by patrick96** [comment]
Yes, I think I see what you mean.

What does polybar actually display when you use two modules?

**Comment by binaryronin** [comment]
![2017-08-01-104437_172x29_scrot](https://user-images.githubusercontent.com/26859485/28831023-824999da-76a6-11e7-946f-47ec0cd06ad4.png)

Polybar isn't doing anything incorrectly, it's just not ideal. I would like to either be able to combine them, or be able to have Polybar switch to showing the percentage when the battery is not in use.

**Comment by patrick96** [comment]
Ah, I see what you mean. I don't think the new label shouldn't be too difficult to implement, but right now I'm very busy. In a few weeks time I may be able to start working on this.

**Comment by binaryronin** [comment]
Awesome! Really appriciate it!

**Comment by FireAntJoe** [comment]
So just to be clear this would fix the need to have double battery modules?

**Comment by patrick96** [comment]
@FireAntJoe No, you would still need to have one module per battery. This feature will only give you a new label type for when the battery is not in use (neither charging nor discharging)

**Comment by Krakn** [comment]
I'll take a whack at this tonight.

**Comment by neh** [comment]
Any chance of this getting picked up again? I'd love to see an Empty option as well as the Unknown, but that might be another issue to open.

**Comment by devopsdeluxe** [comment]
I'll check if I still have my changes lying around anywhere.

If I don't post a link here by Monday, the most accurate method of doing this would be to read the contents of: `/sys/class/power_supply/${BATTERY}${N}` to get the true battery status.

**Comment by devopsdeluxe** [comment]
Found it: https://github.com/devopsdeluxe/polybar/tree/issues/678
